apiserver/facades/agent/caasoperator/mock_test.go

```diff
@@ -44,7 +44,7 @@ func newMockState() *mockState {
 			name: "gitlab",
 			life: state.Alive,
 			charm: mockCharm{
-				url:    charm.MustParseURL("cs:gitlab-1"),
+				url:    charm.MustParseURL("ch:gitlab-1"),
```

Comment:
we don't have revisions in the url anymore, right?

Reply:
We do not allow users to specify ch:<charm>-<revision> to deploy a charm, either on the CLI or inside of a bundle. However we do use it internally to juju.